frontend/src/Components/Login/Login.jsx

```diff
@@ -1,6 +1,151 @@
-import React from "react";
+import React, { useEffect, useRef, useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
 import "./Login.css";
+import logoImage from "./assets/logoblackandwhite.svg";
+import AxiosInstance from "../axios/AxiosInstance";
 
-export const Login = () => {
-  return <div>SkyNest</div>;
+const Login = ({ setAccessToken }) => {
+	const [email, setEmail] = useState("");
+	const [password, setPassword] = useState("");
+	const [errorMsg, setErrorMsg] = useState("");
+	const [showPassword, setShowPassword] = useState(false);
+	const [loading, setLoading] = useState(false);
+	const [successfulLogin, setSuccessfulLogin] = useState("");
+
+	const emailRef = useRef();
+	const passwordRef = useRef();
+
+	const navigate = useNavigate();
+
+	const homePageRedirect = (delay) => {
+		setTimeout(() => {
+			navigate("/home");
+		}, delay);
+	};
+
+	const getUserToken = async () => {
+		await AxiosInstance.post("/login", { email, password })
+			.then(({ data }) => {
+				if (data?.accessToken) {
+					setAccessToken(data.accessToken);
+					localStorage.setItem("accessToken", data.accessToken);
+					setSuccessfulLogin("Login Successful, redirecting.");
+					homePageRedirect(2500);
```

Comment:
Why delay, no need for user to wait 2500ms to gets to homepage if everything went well.

Reply:
Also, try writing async/await fully.

Reply:
Just wanted to give the user a nice visualisation that he logged in successfully with the green success-style text above. 
This is super unnecessary, just looked more visually pleasing.